### PR TITLE
TLUNIZIN-148 Creating a one jar file that has dependencies added to it

### PR DESCRIPTION
**Pull request by pushyamig**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+			</plugin>
+			
+
+
+		</plugins>
+		<resources>
+			<resource>
+				<directory>src/main/resources</directory>
+				<includes>
+					<include>**/*.properties</include>
```
Comment (dlhaines) [inline pom.xml:65]: This includes properties in the build.  How does the tool know the current term?

File: README.md [hunk 2]
```diff
@@ -1 +1,5 @@
 # canvasCourseDeleteUtility
+
```
Comment (dlhaines) [inline README.md:2]: This is not correctly formatted for MarkDown.

This says "Build instructions" but they are both build and run instructions.

Why not have a build.sh?  That is easier to use on the build server.

Tool configuration should be documented.

File: pom.xml (hunk 1 above)
Comment (pushyamig) [inline pom.xml:65]: Current term is not coming from the Property file.

File: README.md (hunk 2 above)
Comment (pushyamig) [inline README.md:2]: The only command right now we have is mvn clean install as part of build, If we put that in build.sh then we are not fully utilizing the Jenkins. I will Make a build.sh but i use it after the build is done and to manipulate the jar naming convention.

**Pull request merged**